src/Symfony/Component/Notifier/Bridge/Brevo/composer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "symfony/brevo-notifier",
+  "type": "symfony-notifier-bridge",
+  "description": "Symfony Brevo Notifier Bridge",
+  "keywords": [
+    "brevo",
+    "notifier"
+  ],
+  "homepage": "https://symfony.com",
+  "license": "MIT",
+  "authors": [
```

Comment:
Please use 4 spaces for indentation, like all other composer.json files in the codebase (and like the indentation used by composer as that's the one used by PHP's `JSON_PRETTY_PRINT`)

Reply:
Good to know